Configuration/TSConfig/Page.ts

```diff
@@ -4,3 +4,13 @@ mod.web_txtemplavoilaplusLayout.enableContentAccessWarning = 1
 mod.web_txtemplavoilaplusLayout.enableLocalizationLinkForFCEs = 0
 mod.web_txtemplavoilaplusLayout.useLiveWorkspaceForReferenceListUpdates = 1
 mod.web_txtemplavoilaplusLayout.adminOnlyPageStructureInheritance = fallback
+
+# example for filtering allowed maps allow single string or array
```

Comment:
This should go in Documentation and not here.
Maybe we should rename the Documentation folder to doc_old and create a new one and fill this up part by part

Reply:
Probably, yes. But this is rather out of scope of the ticket and PR, is it?

Reply:
(and removing the info altogether before a new docs is there is probably not helpful)

Reply:
done via https://github.com/pluspol-interactive/templavoilaplus/pull/343